### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by Balanced02**
- Initialized project
- Initialize git repository
- Setup git branch and merge rules
- Add bootstrap for ease of stylings
- Setup Redux
- Setup routing with react-router-dom
- Setup eslint rules
- Setup README.md and CONTRIBUTIONS guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 - **Project Type:** Open-Source is the new Sauce
 
-- **Estimated Duration:** 3 MONTHS
+- **Estimated Duration:** 4 MONTHS
 
 - **Project Overview:** This project it to be built on [ALOC EXAM ENDPOINTS](https://github.com/Seunope/aloc-endpoints) API which is an API for 5,000 past questions api end-points for POST-UTME, UTME, WASSCE, NECO questions.
```
Comment (samie820) [inline README.md]: just a typo here nothing serious.

File: src/container/Full.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Full.propTypes = {
+  auth: bool.isRequired
+}
+
+const mapStateToProps = state => {
+  return {
+  auth: state.auth.authenticated,
+}}
+
+export default withRouter(connect(mapStateToProps)(Full));
```
Comment (samie820) [inline src/container/Full.jsx]: is there a need to wrap this again...since its a child of the `Router` component?

**Review by inidaname — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Balanced02) [reply README.md]: Thanks... fixed

File: src/container/Full.jsx (hunk 2 above)
Comment (Balanced02) [reply src/container/Full.jsx]: Yeah, there actually was no need... removed already

**Review by samie820 — APPROVED** [review]

**Pull request merged**